.github/workflows/ci.yml

```diff
@@ -241,7 +241,7 @@ jobs:
           fetch-depth: 0
       - uses: coursier/setup-action@v1
         with:
-          jvm: 'temurin:11.0.18'
+          jvm: 'temurin:8'
```

Comment:
So is this the main culprit here? Because we released on 11?

Reply:
I think so, I will test it on a snapshot version